server/http.go

```diff
@@ -134,35 +301,15 @@ func (p *Plugin) handleStartMeeting(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	ru, clientErr := p.zoomClient.GetUser(user.Email)
-	if clientErr != nil {
-		http.Error(w, clientErr.Error(), clientErr.StatusCode)
+	zoomUser, authErr := p.authenticateAndFetchZoomUser(userID, user.Email, req.ChannelID)
```

Comment:
nit: this replicates the original style, but is there a reason not to `s/authErr/err/`?

Reply:
I abstracted the error that may occur during Auth into 
`type AuthError struct {
	Message string `json:"message"`
	Err     error  `json:"err"`
}` which carries different message string for Password based vs OAuth based errors.